Simplenote/src/androidTest/java/com/automattic/simplenote/uitests/setup/SignUpTest.kt

```diff
@@ -0,0 +1,37 @@
+package com.automattic.simplenote.uitests.setup
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.*
+import androidx.test.filters.MediumTest
+import androidx.test.internal.runner.junit4.AndroidJUnit4ClassRunner
+import com.automattic.simplenote.BaseUITest
+import com.automattic.simplenote.R
+import com.automattic.simplenote.utils.Constants
+import com.automattic.simplenote.utils.TestDriver
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4ClassRunner::class)
+@MediumTest
+class SignUpTest : BaseUITest(){
+
+    @Test
+    fun newUserSignUpWithMixedCapitalization() {
+
+        onView(withId(R.id.button_signup))
+            .check(matches(isClickable()))
+            .perform(click())
+
+        onView(withId(R.id.email_edit_text))
+            .check(matches(isClickable()))
+            .perform(click())
+
+        onView(withId(R.id.email_edit_text))
+         .perform(ViewActions.replaceText(Constants.SIGNUP_ACCOUNT_EMAIL), ViewActions.closeSoftKeyboard())
+
+        TestDriver.tapButton(Constants.SIGNUP_BUTTON_STR)
+    }
+}
```

Comment:
Agree. Since this is my Trial period, as part of giving updates, I was under the impression that I had to include everything I had worked upon thus far to show progress, what I am working on, updates etc. and in general showcase my workings and thought process. In a real working scenario though, definitely not something to include in the PR, its not ready just yet.